### PR TITLE
resolved reload after logout and magic link message issues

### PR DESCRIPTION
**Pull request by Mihiirr**

### CODE REVIEW

File: src/app/components/NavBar/ProfileDropdown.tsx [hunk 1]
```diff
@@ -59,6 +58,21 @@ const ProfileDropdown = ({ user, navigationMenu }: Props) => {
...
+                <button
+                  type="submit"
+                  className={classNames(
+                    active ? "bg-gray-100" : "",
+                    "block w-full px-4 py-2 text-left text-sm text-gray-700",
+                  )}
+                >
+                  Sign out
+                </button>
+              </Form>
```
Comment (tirthGajjar) [inline src/app/components/NavBar/ProfileDropdown.tsx:73]: Move this into a separate component.

File: src/app/components/NavBar/ProfileDropdown.tsx [hunk 2]
```diff
@@ -59,6 +58,21 @@ const ProfileDropdown = ({ user, navigationMenu }: Props) => {
               )}
             </Menu.Item>
           ))}
+          <Menu.Item>
+            {({ active }) => (
+              <Form action="/auth/logout" method="post" replace reloadDocument>
+                <button
+                  type="submit"
+                  className={classNames(
```
Comment (tirthGajjar) [inline src/app/components/NavBar/ProfileDropdown.tsx:66]: use clsx
```suggestion
                  className={clsx(
```

**Review by tirthGajjar — CHANGES_REQUESTED** [review]

File: src/app/components/NavBar/ProfileDropdown.tsx (hunk 1 above)
Comment (Mihiirr) [reply src/app/components/NavBar/ProfileDropdown.tsx]: done

File: src/app/components/NavBar/ProfileDropdown.tsx (hunk 2 above)
Comment (Mihiirr) [reply src/app/components/NavBar/ProfileDropdown.tsx]: done

**Review by tirthGajjar — APPROVED** [review]

**Pull request merged**